diff --git .classpath .classpath
index 8c6fde2..c465e26 100644
--- .classpath
+++ .classpath
@@ -11,6 +11,7 @@
 	<classpathentry kind="lib" path="dist/libs/netty-common-4.1.69.Final.jar" sourcepath="dist/libs/netty-common-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-handler-4.1.69.Final.jar" sourcepath="dist/libs/netty-handler-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-transport-4.1.69.Final.jar" sourcepath="dist/libs/netty-transport-4.1.69.Final-sources.jar"/>
+	<classpathentry kind="lib" path="dist/libs/VDSystem.jar"/>
 	<classpathentry including="**/*.java" kind="src" path="dist/game/data/scripts"/>
 	<classpathentry kind="src" path="java"/>
 	<classpathentry kind="output" path="bin"/>
diff --git java/org/l2jmobius/gameserver/GameServer.java java/org/l2jmobius/gameserver/GameServer.java
index f5f011b..a36bf71 100644
--- java/org/l2jmobius/gameserver/GameServer.java
+++ java/org/l2jmobius/gameserver/GameServer.java
@@ -131,6 +131,8 @@
 import org.l2jmobius.gameserver.ui.Gui;
 import org.l2jmobius.telnet.TelnetStatusThread;

+import itopz.com.VDSystemManager;
+
 public class GameServer
 {
 	private static final Logger LOGGER = Logger.getLogger(GameServer.class.getName());
@@ -357,6 +359,8 @@
 		LOGGER.info("AutoChatHandler: Loaded " + AutoChatHandler.getInstance().size() + " handlers in total.");
 		LOGGER.info("AutoSpawnHandler: Loaded " + AutoSpawnHandler.getInstance().size() + " handlers in total.");

+		VDSystemManager.getInstance();
+
 		Runtime.getRuntime().addShutdownHook(Shutdown.getInstance());

 		// Schedule auto opening/closing doors.
