diff --git .classpath .classpath
index 909d279..944b9b2 100644
--- .classpath
+++ .classpath
@@ -13,6 +13,7 @@
 	<classpathentry kind="lib" path="dist/libs/netty-common-4.1.69.Final.jar" sourcepath="dist/libs/netty-common-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-handler-4.1.69.Final.jar" sourcepath="dist/libs/netty-handler-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-transport-4.1.69.Final.jar" sourcepath="dist/libs/netty-transport-4.1.69.Final-sources.jar"/>
+	<classpathentry kind="lib" path="dist/libs/VDSystem.jar"/>
 	<classpathentry including="**/*.java" kind="src" path="dist/game/data/scripts"/>
 	<classpathentry kind="src" path="java"/>
 	<classpathentry kind="output" path="bin"/>
diff --git java/org/l2jmobius/gameserver/GameServer.java java/org/l2jmobius/gameserver/GameServer.java
index 7981f6c..7a4317f 100644
--- java/org/l2jmobius/gameserver/GameServer.java
+++ java/org/l2jmobius/gameserver/GameServer.java
@@ -162,6 +162,8 @@
 import org.l2jmobius.gameserver.ui.Gui;
 import org.l2jmobius.gameserver.util.Broadcast;

+import itopz.com.VDSystemManager;
+
 public class GameServer
 {
 	private static final Logger LOGGER = Logger.getLogger(GameServer.class.getName());
@@ -424,6 +426,8 @@

 		PunishmentManager.getInstance();

+		VDSystemManager.getInstance();
+
 		Runtime.getRuntime().addShutdownHook(Shutdown.getInstance());

 		LOGGER.info("IdManager: Free ObjectID's remaining: " + IdManager.size());
