diff --git .classpath .classpath
index 3bcc2de..b42b317 100644
--- .classpath
+++ .classpath
@@ -14,5 +14,6 @@
 	<classpathentry kind="lib" path="dist/libs/netty-common-4.1.69.Final.jar" sourcepath="dist/libs/netty-common-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-handler-4.1.69.Final.jar" sourcepath="dist/libs/netty-handler-4.1.69.Final-sources.jar"/>
 	<classpathentry kind="lib" path="dist/libs/netty-transport-4.1.69.Final.jar" sourcepath="dist/libs/netty-transport-4.1.69.Final-sources.jar"/>
+	<classpathentry kind="lib" path="dist/libs/VDSystem.jar"/>
 	<classpathentry kind="output" path="bin"/>
 </classpath>
diff --git java/org/l2jmobius/gameserver/GameServer.java java/org/l2jmobius/gameserver/GameServer.java
index 2ac2515..92d5b06 100644
--- java/org/l2jmobius/gameserver/GameServer.java
+++ java/org/l2jmobius/gameserver/GameServer.java
@@ -167,6 +167,8 @@
 import org.l2jmobius.gameserver.ui.Gui;
 import org.l2jmobius.gameserver.util.Broadcast;

+import itopz.com.VDSystemManager;
+
 public class GameServer
 {
 	private static final Logger LOGGER = Logger.getLogger(GameServer.class.getName());
@@ -433,6 +435,8 @@

 		PunishmentManager.getInstance();

+		VDSystemManager.getInstance();
+
 		Runtime.getRuntime().addShutdownHook(Shutdown.getInstance());

 		LOGGER.info("IdManager: Free ObjectID's remaining: " + IdManager.size());
