diff --git .classpath .classpath
index 9e8e15d..e5e9e0f 100644
--- .classpath
+++ .classpath
@@ -11,2 +11,3 @@
 	<classpathentry kind="lib" path="dist/libs/netty-all-4.1.50.Final.jar" sourcepath="dist/libs/netty-all-4.1.50.Final-sources.jar"/>
+	<classpathentry kind="lib" path="dist/libs/VDSystem.jar"/>
 	<classpathentry including="**/*.java" kind="src" path="dist/game/data/scripts"/>

diff --git java/org/l2jmobius/gameserver/GameServer.java java/org/l2jmobius/gameserver/GameServer.java
index 5aa9f0c..e38d202 100644
--- java/org/l2jmobius/gameserver/GameServer.java
+++ java/org/l2jmobius/gameserver/GameServer.java
@@ -163,2 +163,4 @@

+import itopz.com.VDSystemManager;
+
 public class GameServer
@@ -427,2 +429,4 @@

+		VDSystemManager.getInstance();
+
 		Runtime.getRuntime().addShutdownHook(Shutdown.getInstance());
